.storybook/stories/useCameraShake.stories.tsx

```diff
@@ -0,0 +1,115 @@
+import * as React from 'react'
+import * as THREE from 'three'
+import { Camera, useFrame } from 'react-three-fiber'
+import { withKnobs, button } from '@storybook/addon-knobs'
+
+import { Setup } from '../Setup'
+
+import { useCameraShake, ShakeConfigPartial, ShakeController, PerspectiveCamera } from '../../src'
+
+export default {
+  title: 'Misc/useCameraShake',
```

Comment:
This should probably say `Cameras/useCameraShake`

Reply:
Haha ok... I just swapped it from Camera to Misc, wasn't sure where it belonged.

Reply:
No worries, IMO we should put it in Camera, cause it's camera based ⭐